keras/src/backend/torch/layer.py

```diff
@@ -1,39 +1,117 @@
-from typing import Iterator
-from typing import Tuple
-
 import torch
 
-from keras.src.backend.common.stateless_scope import in_stateless_scope
 from keras.src.ops.operation import Operation
 
 
 class TorchLayer(torch.nn.Module):
-    def _post_build(self):
-        # Do not track variables when in a stateless scope.
-        # The variables are not initialized.
-        if in_stateless_scope():
-            return
-        self._track_variables()
-
-    def _track_variables(self):
+    """Adaptation layer to make sure keras.layers.Layer works well with
+    torch.nn.Module. Currently, the main modification are on parameter/module
+    tracking and pointing torch.nn.Module.forward() to the right keras call.
+
+    Module tracking:
+      All sublayers are tracked as modules in Module._modules. All module level
+    api with recurse=True should work properly just like a torch.nn.Module.
+
+    Variable tracking:
+      Since keras has a different variable tracking mechanism, unlike modules,
+    Modules._parameter doesn't automatically tracks variables created for torch
+    layers.
+      This is currently manually populated through _track_torch_params() that
+    does following work:
+        1. Populate all sublayers torch params by calling _track_torch_params()
+        2. Create a single torch.nn.ParameterList() parameter with trainable,
+           non trainable and seed generator states belongs to the current layer.
+
+    Few additional points that user should be aware of:
+    1. When torch backend is enabled KerasVariable.value is torch.nn.Parameter,
```

Comment:
Make sure to introduce a line break (blank line) before a list or after a section title.